server/helpers/Token.js

```diff
@@ -0,0 +1,41 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+
+config();
+/**
+ * @class Token
+ */
+class Token {
+  /**
+   * @static
+   * @param {user} user object from the database
+   * @returns {string} returns thetoken
+   * @memberof Token
+   */
+  static async create(user) {
+    const { id, email } = user;
+    const token = await jwt.sign({
+      userId: id,
+      email,
+    }, process.env.SECRET);
+    return token;
+  }
+
+  /**
+   * @static
+   * @param {*} req the request object
+   * @param {*} res the response object
+   * @param {*} next middleware
+   * @returns {function} next
+   * @memberof Token
+   */
+  static async verifyToken(req, res, next) {
+    const { token } = req.headers;
```

Comment:
please wrap in a `try/catch` block. the `jwt.verify` throws errors on failure

Reply:
I have implemented this.